common/models/role-mapping.js

```diff
@@ -26,9 +27,9 @@ module.exports = function(RoleMapping) {
   RoleMapping.resolveRelatedModels = function() {
     if (!this.userModel) {
       var reg = this.registry;
-      this.roleModel = reg.getModelByType(loopback.Role);
-      this.userModel = reg.getModelByType(loopback.User);
-      this.applicationModel = reg.getModelByType(loopback.Application);
+      this.roleModel = reg.getModelByType('Role');
+      this.userModel = reg.getModelByType('User');
+      this.applicationModel = reg.getModelByType('Application');
     }
```

Comment:
this change is required as the use of `loopback.Role` would lead to inconsistent results given the recent introduction of app-local registry